githubscript/docker.d/worker.yml

```diff
@@ -15,6 +15,12 @@ github_projects:
         github_owner: mozilla-mobile
         github_repo_name: android-components
         contact_github: true
+      focus-android:
```

Comment:
If we want to add a staging-focus-android below (in fake-prod and dev) that may help in some testing scenarios; otherwise we can use the `mock` repo.